recipes/cuda/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "12.0.0" %}
+
+package:
+  name: cuda
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_documentation/LICENSE.txt
+  sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+requirements:
+  run:
+    - cuda-toolkit {{ version }}
+    - cuda-runtime {{ version }}
```

Comment:
I'm confused by the structure here. `cuda` requires `cuda-runtime`, which requires `cuda-libraries`. But `cuda-toolkit` also requires `cuda-libraries`. Why do we need to list `cuda-runtime` here, and why do we need `cuda-runtime` at all?